src/code_7bb60_len_41b0.c

```diff
@@ -10,7 +13,40 @@ void func_800E26B0(void) {
     playerStatus->jumpApexHeight = playerStatus->position.y;
 }
 
-INCLUDE_ASM(s32, "code_7bb60_len_41b0", func_800E26C4);
+s32 func_800E26C4(void) {
+    PlayerStatus* playerStatus = gPlayerStatusPtr;
+    PlayerData* playerData = &gPlayerData;
+    s32 actionState = playerStatus->actionState;
+    s8* temp_8010EBB0 = &D_8010EBB0;
+
+    if (actionState == ActionState_IDLE ||
+        actionState == ActionState_WALK ||
+        actionState == ActionState_RUN ||
+        actionState == ActionState_USE_TWEESTER ||
+        actionState == ActionState_SPIN) {
```

Comment:
can we put the `{` on a newline when the condition is long pretty please
```c
if (actionState == ActionState_IDLE ||
    actionState == ActionState_WALK ||
    actionState == ActionState_RUN ||
    actionState == ActionState_USE_TWEESTER ||
    actionState == ActionState_SPIN)
{
    return 1;
}
```
or even
```c
if (
    actionState == ActionState_IDLE ||
    actionState == ActionState_WALK ||
    actionState == ActionState_RUN ||
    actionState == ActionState_USE_TWEESTER ||
    actionState == ActionState_SPIN
) {
    return 1;
}
```

Reply:
sure, I like the top one